lib/Core/Site/Values/Content.php

```diff
@@ -308,12 +308,12 @@ public function getFieldRelation($fieldDefinitionIdentifier)
 
     public function getFieldRelations($fieldDefinitionIdentifier, $limit = 25)
     {
-        $relations = $this->site->getRelationService()->loadFieldRelations(
+        return $this->site->getRelationService()->loadFieldRelations(
             $this->id,
             $fieldDefinitionIdentifier
+            [],
```

Comment:
Missing a coma in the line above, after `$fieldDefinitionIdentifier`.

Reply:
Fixed in [01ae43](https://github.com/netgen/ezplatform-site-api/pull/68/commits/01ae43193b234df92a60bfa5c301525934651584)